<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-MRVJBNT" failures="0" tests="1" name="com.menumitratCommonAPITestScript.verifyOTPTestScript" time="0.002" errors="1" timestamp="22 May 2025 18:35:33 GMT">
  <testcase classname="com.menumitratCommonAPITestScript.verifyOTPTestScript" name="verifyOTPUsingValidInputData" time="0.002">
    <error message="Error during verify OTP test: JSONObject[&quot;mobile_number&quot;] not found." type="java.lang.Exception">
      <![CDATA[java.lang.Exception: Error during verify OTP test: JSONObject["mobile_number"] not found.
	at com.menumitratCommonAPITestScript.verifyOTPTestScript.verifyOTPUsingValidInputData(verifyOTPTestScript.java:163)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- verifyOTPUsingValidInputData -->
</testsuite> <!-- com.menumitratCommonAPITestScript.verifyOTPTestScript -->
